### ISSUE TITLE
ZINCRBY member should be value-typed

### ISSUE THREAD

**Issue by fuyuanpai** [lettuce-io/lettuce-core#826]
## Bug Report

In RedisSortedSetCommands or RedisSortedSetAsyncCommands

The method:
```java
zincrby(K key, double amount, K member);
```
should be 
```java
zincrby(K key, double amount, V member);
```

**Comment by mp911de** [comment]
Good catch. We will fix that with the next minor release 4.5 and 5.1.

**Comment by mp911de** [comment]
That's fixed now.
